readthedocs/templates/builds/build_detail.html

```diff
@@ -161,6 +162,11 @@ <h3>{% trans "Error" %}</h3>
            data-bind="text: error">
           {{ build.error }}
         </p>
+        <p>
+        {% blocktrans with url=issue_url %}
+         Report any build issues <a href="{{ url }}">here</a>.
```

Comment:
This should be surrounded in a template block so that we can avoid this block on our commercial hosting site. We don't want github issues from build errors there.

Reply:
@agjohnson I didn't get this one. Can you just make this a bit more clear?  Your reason is perfectly fine, I agree with that too. But I didn't get how to do this one. Sorry I may be not familiar with this one.

Reply:
Sorry, I should have clarified more here. You'll want to use a named block, something like:

```
{% block github_issue_link %}
  {% blocktrans %}
    ...
  {% endblocktrans %}
{% endblock %}
```

That way, we can override this on our commercial hosting and avoid a link to GitHub.

Reply:
Okay Thanks !